pcdsdevices/lasers/rfof.py

```diff
@@ -11,19 +12,628 @@
+        kind="normal",
+        doc="Fan revolutions are too low"
+    )
+    pwr_supply_i_err = Cpt(
+        EpicsSignalRO,
+        ":PWR_SUPPLY_I_ERR",
+        kind="normal",
+        doc="Power supply currents are too high"
+    )
+    tx_eth_err = Cpt(
+        EpicsSignalRO,
+        ":TX_ETH_SYNC_ERR",
+        kind="normal",
+        doc=(
+            "Tx: Ethernet sync between units "
+            "lost for more than 10 min"
+        )
+    )
+    rx_eth_err = Cpt(
+        EpicsSignalRO,
+        ":RX_ETH_SYNC_ERR",
+        kind="normal",
+        doc=(
+            "Rx: Ethernet sync between units "
+            "lost for more than 10 min"
+        )
+    )
+    tx_input_err = Cpt(
+        EpicsSignalRO,
+        ":TX_RF_INPUT_ERR",
+        kind="normal",
+        doc="RF input is out of range for more than 60s"
+    )
+    rx_output_err = Cpt(
+        EpicsSignalRO,
+        ":RX_RF_OUTPUT_ERR",
+        kind="normal",
+        doc="RF output is out of range for more than 10 min"
+    )
+    tx_temp_sens_err = Cpt(
+        EpicsSignalRO,
+        ":TX_TEMP_SENS_ERR",
+        kind="normal",
+        doc=(
+            "Tx: Temperature sensors are not working properly; "
+            "TEC protection activated"
+        )
+    )
+    rx_temp_sens_err = Cpt(
+        EpicsSignalRO,
+        ":RX_TEMP_SENS_ERR",
+        kind="normal",
+        doc=(
+            "Rx: Temperature sensors are not working properly; "
+            "TEC protection activated"
+        )
+    )
+    tx_rf_idn_err = Cpt(
+        EpicsSignalRO,
+        ":TX_RF_IDN_ERR",
+        kind="normal",
+        doc="Tx: RF control identification failed"
+    )
+    tx_rf_sens_err = Cpt(
+        EpicsSignalRO,
+        ":TX_RF_SENS_ERR",
+        kind="normal",
+        doc="Tx: RF control sensor failure"
+    )
+    tx_arm_idn_err = Cpt(
+        EpicsSignalRO,
+        ":TX_ARM_IDN_ERR",
+        kind="normal",
+        doc="Tx: Arm control identification failed"
+    )
+    tx_arm_sens_err = Cpt(
+        EpicsSignalRO,
+        ":TX_ARM_SENS_ERR",
+        kind="normal",
+        doc="Tx: Arm control sensor failure"
+    )
+    unlock_err = Cpt(
+        EpicsSignalRO,
+        ":UNLOCK_ERR",
+        kind="normal",
+        doc="System is unlocked for more than 15 min"
+    )
+    rx_rf_idn_err = Cpt(
+        EpicsSignalRO,
+        ":RX_RF_IDN_ERR",
+        kind="normal",
+        doc="Rx: RF control identification failed"
+    )
+    rx_rf_sens_err = Cpt(
+        EpicsSignalRO,
+        ":RX_RF_SENS_ERR",
+        kind="normal",
+        doc="Rx: RF control sensor failure"
+    )
+    rx_arm_idn_err = Cpt(
+        EpicsSignalRO,
+        ":RX_ARM_IDN_ERR",
+        kind="normal",
+        doc="Rx: Arm control identification failed"
+    )
+    rx_arm_sens_err = Cpt(
+        EpicsSignalRO,
+        ":RX_ARM_SENS_ERR",
+        kind="normal",
+        doc="Rx: Arm control sensor failure"
+    )
+
+
+class ItechRfofBase(BaseInterface, Device):
```

Comment:
Quick check: why is this called `Base`? I'm usually used to seeing `Base` in class names when they are parent classes for other classes to inherit from, but that's not the case here.  Is there some device-specific thing called a "base" that this represents?

You may consider a short docstring here explaining what this device is, so the next person who browses pcdsdevices understands what this device is for.

Reply:
You're right. Perhaps, just naming it "ItechRfof" by itself?

Reply:
`ItechRfof` is totally :+1:

Reply:
The name in your previous diff is totally good too- just want to make sure we've considered the name since it was confusing me.

If we change the name, we also need to change the import in `pcdsdevices.device_types`